specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/definitions.json

```diff
@@ -9232,13 +9232,60 @@
         "description": {
           "type": "string",
           "description": "Description of the workspace."
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of workspace.",
```

Comment:
As discussed in the spec, their should not be any `kind` property on the Workspace. The Workspace inherits the kind of the Gateway to which it is attached to. Please revert

Reply:
No, this is what was agreed on - Workspace gets tainted with the kind from the first gateway which it gets tainted with.

This kind is a readonly property for UI purposes and for customers to know what it was tainted with in the past. If we don't offer this information, then the only way for customers to know, other than portal, is to get list of associated gateways and check respective kinds.

Reply:
Discussed offline, removing the `kind` from the workspace.